watertap/unit_models/tests/test_uv_aop.py

```diff
@@ -192,7 +206,7 @@ def test_initialize(self, UV_frame):
     def test_var_scaling(self, UV_frame):
         m = UV_frame
         badly_scaled_var_lst = list(badly_scaled_var_generator(m))
-        [print(i[0]) for i in badly_scaled_var_lst]
+        [print(i[0].name, i[1]) for i in badly_scaled_var_lst]
```

Comment:
I don't have a problem with this line being here, but we typically delete these print lines since they are meant for debugging purposes and we usually clean this kind of stuff up before merging.

Reply:
Deleted